scripts/errors/generate-errors.js

```diff
@@ -14,29 +14,19 @@ for (api in apis) {
     exit(1);
   }
 
-  stringBuffer = `
-/**
-   ${api} error codes
-
-    \`\`\`ts
-    apiCall.then((data) => {
-        console.log(data?.data.items)
-    }).catch((err) => {
-      if(APIErrorCodes.ERROR_5 == err.response?.data.code) {
-        // Handle error
-      }
-    })
-    \`\`\`
-*/
-export const APIErrorCodes = {
+  stringBuffer = 
+`from enum import Enum 
+
+#   ${api} error codes
+
+class APIErrorCodes(Enum):
```

Comment:
This creates a python Enum. from reading an enum appears to be the preferred structure to hold errors as there are no consts in python. 

As it stands though this enum doesn't appear to be used. I'm not sure how the SDK incorporates them. 

any knowledge @wtrocki ?

Reply:
Yes. All we need is to import it and use it's value. 
See: 
https://github.com/redhat-developer/app-services-sdk-go/tree/main/registrymgmt#error-handling